conan/tools/cmake/cmake.py

```diff
@@ -304,11 +309,23 @@ def ctest(self, cli_args=None, env="", stdout=None, stderr=None):
         if njobs:
             arg_list.append(f"--parallel {njobs}")
 
+        arg_list.extend(cli_args or [])
+
         verbosity = self._conanfile.conf.get("tools.build:verbosity", choices=("quiet", "verbose"))
         if verbosity:
+            # https://cmake.org/cmake/help/latest/manual/ctest.1.html
+            # Options such as --verbose, --extra-verbose, and --debug are
+            # ignored if --quiet is specified.
+            arg_list = [a for a in arg_list if a not in ("--quiet", "--verbose")]
             arg_list.append(f"--{verbosity}")
```

Comment:
not related directly with this PR, but checking this:

```
arg_list.append(f"--{verbosity}")
```

can't this break in the future if we extend the conf to accept other choices than "quiet", "verbose" and add an unsupported argument to ctest?